packages/lib/src/components/internal/Await/Await.tsx

```diff
@@ -67,7 +80,7 @@ function Await(props: AwaitComponentProps) {
             .catch(({ message, ...response }) => ({
                 type: 'network-error',
                 props: {
-                    ...(message && { message: this.props.i18n.get(message) }),
+                    ...(message && { message: props.getI18n(message) }),
```

Comment:
Why was this changed?

Reply:
Because we don't actually use it.
And if we were to use it I thought we should just pass in a function that retrieves a translation rather than the translation "machine" itself (i.e. `i18n`) - which means the Await comp then needs to know about the existence of things like a `get` function

Reply:
My mistake - I didn't think it ever received a `i18n` prop - but I see it retrieves it directly from `CoreContext`.
I'll change it